brave/RATIONALE.md

```diff
@@ -60,6 +60,15 @@ attempting to do that would limit the applicability of Brave, which is an anti-
 goal. Instead, we choose to use nothing except floor Java version features,
 currently Java 6.
 
+Here's an example of when things that seem right aren't. We once dropped our
+internal `@Nullable` annotation (which is source retention), in favor of JSR
+305 (which is runtime retention). In doing so, we got `null` analysis from
```

Comment:
you can still get `null` analysis from intellij if you add your custom annotation here 
![image](https://user-images.githubusercontent.com/1935500/81564708-111e6600-9390-11ea-8171-8590b26e5b91.png)

Reply:
Thanks @NersesAM. I just updated with this.